bin/suitcss

```diff
@@ -65,7 +65,13 @@ program.parse(process.argv);
 
 var input = program.args[0] ? resolve(program.args[0]) : null;
 var output = program.args[1] ? resolve(program.args[1]) : null;
-var config = program.config ? require(resolve(program.config)) : null;
+var config;
+var configFile = program.config;
+if (configFile && configFile.endsWith('.js')) {
+  config = require(resolve(configFile));
+} else if (configFile) {
+  config = fs.readJsonSync(resolve(configFile));
+}
```

Comment:
@simonsmith alright, I factored the fn into `bin/suitcss` again. 
also I left out unit tests for now, as the fns scope is covered by the integration tests already. good?